h/models/group.py

```diff
@@ -34,6 +34,18 @@ class WriteableBy(enum.Enum):
     members = "members"
 
 
+class GroupMembership(Base):
```

Comment:
I think all the other model classes are named after their DB tables so this would be `UserGroup`. But `GroupMembership` is nicer so :man_shrugging:

Reply:
I didn't even think about it, it's just what I've been calling it all over the shop in Bulk API.